src/service/performance-impl.js

```diff
@@ -92,8 +92,16 @@ export class Performance {
     /** @private {boolean} */
     this.isPerformanceTrackingOn_ = false;
 
-    /** @private {?string} */
-    this.enabledExperiments_ = null;
+    /** @private {!Object<string,boolean>} */
+    this.enabledExperiments_ = map();
+    /** @private {string} */
+    this.ampexp_ = '';
+
+    // Add RTV version as experiment ID, so we can slice the data by version.
+    this.addEnabledExperiment(getMode(this.win).rtvVersion);
```

Comment:
good suggestion. i can do it in a separate PR since this is already in prod, and it will change prod behavior.